frontend/src/component/Header/MenuActionListItem/NotificationButton.jsx

```diff
@@ -13,11 +13,17 @@ export const NotificationButton = props => {
       >
         <i className='far fa-fw fa-bell' />
         {props.t('Notifications')}
-        {props.notificationNotReadCount > 0 && (
+        {props.unreadMentionCount > 0 && (
           <div
-            className='notificationButton__count'
+            className='notificationButton__mention'
+          >
+            {props.unreadMentionCount > 99 ? '99+' : props.unreadMentionCount}
+          </div>
+        )}
+        {props.unreadMentionCount === 0 && props.unreadNotificationCount > 0 && (
+          <div
+            className='notificationButton__notification'
           >
-            {props.notificationNotReadCount > 99 ? '99+' : props.notificationNotReadCount}
           </div>
```

Comment:
this div can be a self-closing tag
`<div className='notificationButton__notification' />`